### ISSUE TITLE
are you sure? I use 3090 when the batch-size is 2, the GPU memory is full directly

### ISSUE THREAD

**Issue by Jacky-Android** [xhu248/AutoSAM#3]
I just want to ask what is your experimental configuration.

**Comment by xhu248** [comment]
yes, I just used one V100 with 16GB memory, notice that our default setting is vit-b instead of vit-h, and we freeze the weights in the encoder.

**Comment by Jacky-Android** [comment]
> yes, I just used one V100 with 16GB memory, notice that our default setting is vit-b instead of vit-h, and we freeze the weights in the encoder.

Thank you for your answer, it inspired me a lot. I would like to ask about doing semantic now
